### ISSUE TITLE
A11y_Model Builder_Forecasting_Data_AI4D: (msedgewebview2/region '') An on-screen element must not have a null BoundingRectangle property.

### ISSUE THREAD

**Issue by Samhitha66** [dotnet/machinelearning-modelbuilder#2666]
### GitHubTags

#WCAG4.2.1; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Object Information; #Win32; #Benchmark; #AI4D;

### Environment Details

Application Name: Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621

### Repro steps

1. Launch the Visual Studio with at least one of the .NET workload.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu(Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear.
8. Tab till "Forecasting" and press enter.
9. Setup environemnt screen will appear, Tab till Local CPU tile and select it.
10. Tab till Next button and press enter.
11. Data screen will appear.
12. Turn on AI4D
13. Verify whether we are getting AI bug like An on-screen element must not have a null BoundingRectangle property or not.

The following accessibility issue needs investigation.

### App

msedgewebview2

### Element path

 region ''

### Issue Details

An on-screen element must not have a null BoundingRectangle property. [Section 508 502.3.1](https://www.access-board.gov/ict/#502-interoperability-assistive-technology)

### How To Fix

If the element is off-screen, set its IsOffscreen property to TRUE.
If the element is on-screen, provide a BoundingRectangle property.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

### Attachment
![Screenshot (791)](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/b6d4490f-cea3-45a8-bfba-d61b7ac707f6)

**Comment by zewditu** [comment]
This is not a traditional WPF element we can control its behavior, hence it is expected to fail accessibility tests.

**Comment by Samhitha66** [comment]
@zewditu 
Is this external to any different service? 
If yes please let us know so that we will route this bug.
#Regressed:07-17-23; #Pendinginput;

**Comment by zewditu** [comment]
@Samhitha66 , One more thing, I am not able to reproduce "an on-screen element must not have rounding rectangle" in our latest version.  For the case of "chromium components should be scanned with a web based scanner " this does not apply for is because this app  windows app 
![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/36615490/1a676ba7-5f67-41a6-b395-586bf0ee0953)
